app/Plugin/EnvSource/Lib/lang.php

```diff
@@ -42,7 +42,11 @@
     EnvSourceDuplicateModeEnum::AnyIdentifier    => 'Any Identifier Match',
     EnvSourceDuplicateModeEnum::LoginIdentifier  => 'Login Identifier Match',
   ),
-  
+
+  'pl.envsource.en.auth.provider' => array(
+    AuthProviderEnum::Shibboleth    => 'Shibboleth',
```

Comment:
For pedantic reasons, this should probably say "Shibboleth SP".

Reply:
Done